boards/arm/wio_lte_cat_1/pinmux.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2020 Kieran Bingham <kbingham@kernel.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <drivers/pinmux.h>
+#include <sys/sys_io.h>
+
+#include <pinmux/stm32/pinmux_stm32.h>
+
+/* pin assignments for Wio LTE Cat.1 board */
+static const struct pin_config pinconf[] = {
+#ifdef CONFIG_UART_1
+	{ STM32_PIN_PB6, STM32F4_PINMUX_FUNC_PB6_USART1_TX },
+	{ STM32_PIN_PB7, STM32F4_PINMUX_FUNC_PB7_USART1_RX },
+#endif  /* CONFIG_UART_1 */
+#ifdef CONFIG_UART_2
+	{ STM32_PIN_PA2, STM32F4_PINMUX_FUNC_PA2_USART2_TX },
+	{ STM32_PIN_PA3, STM32F4_PINMUX_FUNC_PA3_USART2_RX },
+#endif  /* CONFIG_UART_2 */
+#ifdef CONFIG_UART_6
```

Comment:
I'm not seeing UART_6 being introduced in Kconfig or dts. Can you cross check?

Reply:
Ah, yes this is likely my debug serial. Can likely be removed and kept in my local patch.